### PR TITLE
A few optimizations and documentation fixes

### PR DESCRIPTION
**Pull request by reugn**

### CODE REVIEW

File: sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/export/CollectionHandle.java [hunk 1]
```diff
@@ -37,15 +38,13 @@ private CollectionHandle(int index) {
...
+   * Construct a new (mutable) set consisting of the passed in collection handles.
    *
    * <p>Used by tests.
    */
   static Set<CollectionHandle> of(CollectionHandle... handles) {
     Set<CollectionHandle> result = mutableSet();
-    for (CollectionHandle handle : handles) {
-      result.add(handle);
-    }
+    result.addAll(Arrays.asList(handles));
```
Comment (anuraaga) [inline sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/export/CollectionHandle.java:47]: Can you revert this one? Doesn't seem to be an obvious improvement

**Review by anuraaga — COMMENTED** [review]
Thanks for the cleanups

File: sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/DefaultAggregation.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-/** Aggregation that selects the specified default based on insturment. */
+/** Aggregation that selects the specified default based on instrument. */
```
Comment (anuraaga) [inline sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/DefaultAggregation.java:15]: Actually it would be good to separate PRs for typo fixes and actual logic changes if that's OK

File: sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/export/CollectionHandle.java (hunk 1 above)
Comment (reugn) [reply sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/export/CollectionHandle.java]: @anuraaga reverted.

File: sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/DefaultAggregation.java (hunk 2 above)
Comment (anuraaga) [reply sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/DefaultAggregation.java:15]: Thanks @reugn - as this is small let me go ahead and merge, in the future would be good to split cosmetic and logic changes.

**Review by anuraaga — APPROVED** [review]
